src/openforms/submissions/tests/test_post_submission_event.py

```diff
@@ -1023,6 +1029,76 @@ def test_payment_complete_does_not_set_retry_flag(self):
 
             self.assertFalse(submission.needs_on_completion_retry)
 
+    @freeze_time("2024-02-16T18:15:00Z")
```

Comment:
I finally used the zgw plugin since I wanted to be able to mock and test the `pre_register` method